mentions/src/test/kotlin/io/spine/examples/pingh/mentions/GitHubClientSpec.kt

```diff
@@ -82,9 +82,7 @@ internal class GitHubClientSpec : ContextAwareTest() {
 
     @BeforeEach
     internal fun prepareSessionsContextAndEmitEvent() {
-        gitHubClientService.unfreeze()
-        gitHubClientService.setDefaultResponseStatusCode()
-        gitHubClientService.mentionsAreNotFetched()
+        gitHubClientService.reset()
```

Comment:
Is it still a "service"? I though we had renamed its base type to `GitHubSearch`.